docs/how-to/mount-block-device.rst

```diff
@@ -171,3 +171,36 @@ Ceph cluster state post IO:
 Comparing the ceph status output before and after writing the file shows that
 the MicroCeph cluster has grown by 30MiB which is thrice the size of the file
 we wrote (10MiB). This is because MicroCeph configures 3 way replication by default.
+
+RBD features and older clients
+------------------------------
+
+MicroCeph sets the default RBD features to 63 which corresponds to ``layering + exclusive-lock + object-map + fast-diff + deep-flatten + stripingv2``. This is done to ensure better performance and features for the RBD images. However, older kernels might not support all these features.
+
+If you face issues mapping RBD images on older kernels, you can either disable the unsupported features for the specific image or change the default behaviour for the cluster.
+
+To disable unsupported features for a specific image:
+
+.. code-block:: none
+
+    $ sudo rbd feature disable block_pool/bd_foo object-map fast-diff deep-flatten
+
+To change the default behaviour for the cluster, you can update the ``rbd_default_features`` config option:
```

Comment:
Here (and in a few sections below), I'd suggest a bit more authoritative language, i.e. remove the "you can" to give users more confidence and direction/not feel like there are other options to consider (unless there are).